firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/GeoPoint.kt

```diff
@@ -1,5 +1,6 @@
 package dev.gitlive.firebase.firestore
 
+/** A class representing a Firebase GeoPoint. */
```

Comment:
I think a better way would be to declare a wrapper object for geo points. this would allow
1. to specify a serializer so the end user won't have to specify it everywhere
2. would allow equals to use JS implementation (called `isEqual`)

but it'd have to wait for a follow up PR